pkg/universe.dagger.io/python/image.cue

```diff
@@ -0,0 +1,26 @@
+package python
+
+import (
+	"universe.dagger.io/docker"
+)
+
+#Image: {
+	// The python version to use
+	version: *"3.10" | string
+
+	// Whether to use the alpine-based image or not
+	alpine: *true | false
+
+	_tag: {
+		if alpine {
+			"\(version)-alpine"
+		}
+		if !alpine {
+			version
+		}
+	}
+
+	docker.#Pull & {
+		source: "python:\(_tag)"
+	}
+}
```

Comment:
Nit: You can use a disjunction instead of an if statement by replacing `#Image` with the one below
```cue
#Image: {
	// The python version to use
	version: *"3.10" | string

	// Whether to use the alpine-based image or not
	alpine: *true | bool

	// Source is define through the disjunction
	docker.#Pull

	*{
		alpine: true

		source: "python:\(version)-alpine"
	} | {
		alpine: false

		source: "python:\(version)"
	}
}
```

Reply:
LGTM. Change added and tests still pass for me locally. Thanks for the help!